src/stavenote.js

```diff
@@ -725,6 +739,7 @@ export class StaveNote extends StemmableNote {
   setKeyLine(index, line) {
     this.keyProps[index].line = line;
     this.note_heads[index].setLine(line);
+    this.reset();
```

Comment:
And actually the line above it probably needs to be removed (and `setLine` does nothing other than set a `line` property)